tests/test_util/test_table.py

```diff
@@ -1280,6 +1280,12 @@ def test_head(self):
         self.assertEqual(head.data.shape[0], 3)
         self.assertEqual(len(head.output.splitlines()), 9)
         self.assertEqual(head.data.tolist(), self.t1_rows[:3])
+        # tests when number of rows < default
+        t = make_table(data=dict(a=["a"], b=["b"]))
+        t.head()
```

Comment:
these calls will be writing to stdout. Might be wise to redirect them to null by putting this in a content manager. See if pytest has a function to help with this.